orttraining/orttraining/python/training/ortmodule/_custom_autograd_function_exporter.py

```diff
@@ -165,6 +165,14 @@ def _export(g, n, *args, **kwargs):
         raise wrap_exception(ORTModuleONNXModelException, e)
 
 
+try:
```

Comment:
Please add a comment explaining why this is needed and when it will be safe to remove the try / except (I guess once the earliest version of PyTorch that's supported is 1.11).